README.md

```diff
@@ -0,0 +1,58 @@
+The main reason why we use Linux is to have a similar environment to the Raspberry Pi.
+Another reason why we use Linux is because our software only works on Linux right now.
+We recommend two options if you do not have Linux already.
+
+# Dual booting [Highly Recommended] 
+
+## Background Knowledge 
+--------------- 
+Dual booting will keep 2 operating systems on your machine (only one operating system will be running on your machine at a time), so when you first start your computer, you can choose which operating system you want to run on.
+We highly recommend dual booting since you can utilize 100% of your computer's hardware for only one operating system(using a Virtual Machine, shares the PC hardware between operating systems; see VM Section).
+
+Pros | Cons
+------------- | -------------
+computer's performance is maximized since it only needs to focus on the one operating system  | Switching between operating systems requires you to reboot the computer
+
+For both dual booting and installing Linux on a VM, get a Linux distribution of your choice.
+   ### Recommended distros are
+
+   	* Linux Mint (Latest Version)
+   	* Ubuntu (Latest Version)
+
+## How to Dual Boot
+--------------- 
+Follow the instructions and/or watch the videos below.
+
+**For Windows**
+Written Instructions: https://itsfoss.com/install-ubuntu-1404-dual-boot-mode-windows-8-81-uefi/ 
+Video: https://www.youtube.com/watch?v=MSVV_EoApdo
+
+**For Macs**
+Written Instructions: https://www.howtogeek.com/187410/how-to-install-and-dual-boot-linux-on-a-mac/
+Video: https://www.youtube.com/watch?v=kRgKlcm1XPI
+
+
+# Running Linux in a Virtual Machine 
+
+## Background Knowledge 
+A virtual machine allows two operating systems to run simultaneously (Sharing the computer's hardware).
+Using a virtual machine separates the host machine and the simulated machine, so if you are running linux in the virtual machine and windows as your main OS, they will have no idea that each other exist.
+
+Pros | Cons
```

Comment:
Another few pros I can think of for the virtual machine is that if you get a virus in the guest OS(linux), then the host OS (ie windows) is still safe.
 
Setup is easier too, and there's no chance of bricking your computer